dbt-get-started/README.md

```diff
@@ -56,12 +56,50 @@ dbt run
 
 > :crab: As an exercise, you can add models for the queries demonstrating [joins](https://materialize.com/docs/get-started/#joins) and [temporal filters](https://materialize.com/docs/get-started/#temporal-filters).
 
+### Test the project
```

Comment:
@ahelium, after shuffling things around, I think it makes more sense to jump straight to continuous testing, rather than adding to the old text. Does it make sense? I wonder if there are scenarios where users will _not_ want to run testing continuously. 🤔 

Reworded the original a bunch and tried to use some terminology we want to standardize on (_e.g._ continuous testing). Let me know if something feels yucky.

Reply:
Sure! Go for it.